utils/hare-status

```diff
@@ -238,13 +238,13 @@ def main(argv=None):
     opts = parse_opts(argv)
     cns = Consul()
     if not cluster_online():
-        print('Cluster is not running.', file=sys.stderr)
-        return 1
+        print('Cluster is not running', file=sys.stderr)
+        return
 
     if opts.json:
         status = get_cluster_status(cns)
         print(j.dumps(status, indent=2, cls=FidEncoder))
-        return 0
+        return
```

Comment:
Suggestion:

```python
if __name__ == '__main__':
    try:
        sys.exit(main())
    except HAConsistencyException as err:
        print(f'Consul Error: {err}')
        code = 1 if fail_on_consul_issue else 0
        sys.exit(code)
    except Exception as e:
        sys.exit('Unexpected problem appeared: {e}')
```

Reply:
Unfortunately, Github doesn't allow commenting lines that are outside of the diff (although they may be related to the fix being discussed, huh)

Reply:
@knekrasov: you can express your opinion about the GitHub issue at https://jts.seagate.com/browse/EOS-11596.

Reply:
@max-seagate It is closed with "Can't fix this issue." 

Why don't we close our cortx issues with the same explanation? 😄